shopfloor/models/shopfloor_menu.py

```diff
@@ -140,6 +140,8 @@ class ShopfloorMenu(models.Model):
     no_prefill_qty_is_possible = fields.Boolean(
         compute="_compute_no_prefill_qty_is_possible"
     )
+    allow_rma_is_possible = fields.Boolean(compute="_compute_allow_rma_is_possible")
+    allow_rma = fields.Boolean(string="Allow create returns", default=False, help="")
```

Comment:
Best help ever ;)

Reply:
When enabled, you can receive unplanned products that are returned from an existing delivery matched on the origin (SO name)

Reply:
I don't even know if it makes sense…

Reply:
Yes, it was a good idea. Nevertheless, whatever scenario implements it, the intent is that one